### PR TITLE
chore: add event logging to maps type

### LINKED ISSUE #1385: Maps: Event Logging for requesting MapAPI
- [x] dataexplorer.maps.viewswitcherdropdown.select_map_type: ViewSwitcherDropdown where users select a map
- [x] maps.mapviewtype.map_token_request.success: In Viewing maps, we make a request to get map token. When request is a success we want to log an event for success.
- [x] maps.mapviewtype.map_token_request.failure: In Viewing maps, we make a request to get map token. When request is failed, we want to log an event for failure.
- [x] maps.mapviewtype.get_geo_coordinates.success: In Viewing maps, we search the data for coordinate columns. When function is executed, we want to log an event for success.
- [x] maps.mapviewtype.get_geo_coordinates.failure: In Viewing maps, we search the data for coordinate columns. When function throws an error, we want to log an event for failure.

### PR DESCRIPTION
**Pull request by Palakp41**
Closes #1385

### CODE REVIEW

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 43 | :red_circle:
accessibility | 72 | :yellow_circle:
seo | 67 | :yellow_circle:
pwa | null | :hankey:
best-practices | 75 | :yellow_circle:


[Compare your scores with master branch](true1387)

File: src/visualization/types/Map/view.tsx [hunk 1]
```diff
@@ -47,8 +48,10 @@ const GeoPlot: FC<Props> = ({result, properties}) => {
         const {token} = await getMapToken()
         setMapToken(token)
         setMapServiceError(RemoteDataState.Done)
+        event('maps.mapviewtype.map_token_request.success')
```
[the remark this answers is not in the document]
Comment (Palakp41) [reply src/visualization/types/Map/view.tsx]: Updated it to include your suggestion :)

**Review by hoorayimhelping — APPROVED** [review]
Nice work Palak!! Looks good to me, but you might want to run these by @kristinarobinson to make sure the strings work for her

**Comment by Palakp41** [comment]
I asked @kristinarobinson to review, she has given me her approval via DM.

**Pull request merged**